### PR TITLE
docs(All): add contribution docs

### LINKED ISSUE #329: add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by christianalfoni**
resolves #329 
First iteration, please review :)

### CODE REVIEW

**Comment by Guria** [comment]
Looks good for first iteration

File: README.md [hunk 1]
```diff
@@ -15,6 +15,31 @@ A state controller with its own debugger
 ## Please head over to our WIP website for cerebral v2
 [http://www.cerebraljs.com/](http://cerebral.github.io/cerebral-website). You will find all the information you need there.
```
Comment (Guria) [inline README.md]: text of link should be changed

File: README.md [hunk 2]
```diff
@@ -15,6 +15,31 @@ A state controller with its own debugger
...
+`npm test`
+
+Or you can run tests for specific packages by going to package root and do the same:
+
+`npm test`
+
+### Changing the code
+When you make a code change you should create a branch first. When the code is changed and backed up by a test you can commit it from **the root** using:
+
+`git cz`
```
Comment (Guria) [inline README.md]: npm run commit

File: README.md [hunk 3]
```diff
@@ -15,6 +15,31 @@ A state controller with its own debugger
 ## Please head over to our WIP website for cerebral v2
 [http://www.cerebraljs.com/](http://cerebral.github.io/cerebral-website). You will find all the information you need there.
 
+## Contribute
+The entire Cerebral codebase has been rewritten to encourage contributions. The code is cleaned up, commented and all code is in a "monorepo". That means you can run tests across projects and general management of the code is simplified a lot.
+
+1. Clone the monorepo
+2. In root: `npm install` (Do not use YARN)
```
Comment (Guria) [inline README.md]: we can omit yarn mention here. nothing wrong in installing monorepo deps with yarn. but lerna will use npm while bootstrap anyway

File: package.json [hunk 4]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "cerebral-monorepo",
   "scripts": {
+    "configure": "lerna bootstrap && npm run build",
```
Comment (Guria) [inline package.json]: `lerna bootstrap` is enough. it runs prepublish scripts for every package.

File: README.md (hunk 2 above)
Comment (edgesoft) [inline README.md]: Run `npm run configure` which will build code and bootstrap it together. Shouldn't it be npm run setup

File: package.json (hunk 4 above)
Comment (Guria) [inline package.json]: there is npm run setup in master already

File: README.md [hunk 5]
```diff
@@ -18,6 +18,34 @@ A state controller with its own debugger
 ## Cerebral v2 is still a work in progress. Please head over to our cerebral v2 WIP website for more information:
 [https://cerebral.github.io/cerebral-website/](http://cerebral.github.io/cerebral-website).
 
+## Contribute
+The entire Cerebral codebase has been rewritten to encourage contributions. The code is cleaned up, commented and all code is in a "monorepo". That means you can run tests across projects and general management of the code is simplified a lot.
+
+1. Clone the monorepo
+2. In root: `npm install`
+3. Run `npm run configure` which will build code and bootstrap it together
```
Comment (Guria) [inline README.md]: configure -> setup

File: package.json [hunk 6]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "scripts": {
     "setup": "lerna bootstrap",
+    "commit": "git-cz",
```
Comment (Guria) [inline package.json]: commit script is already exists

**Pull request merged**